src/app_main.c

```diff
@@ -47,49 +47,32 @@ void app_main() {
     explicit_bzero(&G_context, sizeof(G_context));
```

Comment:
Do we remove this also ?

Reply:
It doesn't cost much, and since we just recently change the behavior when an error is Thrown from calling in a loop app_main to exiting the app, I'd say we should keep it.